sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -726,6 +743,7 @@ SUBROUTINE MDLFLD
 
         DO L=1,LM
           LL=LM-L+1
+          IF(T(I,J,LL)<spval)THEN
```

Comment:
add check PMID in line 746?
@JesseMeng-NOAA

Reply:
will do

Reply:
done